try {
    throw new Error("terribly sorry");
} catch (blunder) {
    inform("A blunder occurred:", blunder.message);
}